cloud-gallery/.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
no rules at all?

Reply:
They are all covered by `.editorconfig`.

Reply:
ah I had no idea that prettier would pick that up! 🙂 

in the [prettier docs](https://prettier.io/docs/en/configuration.html#editorconfig) they say that you need to add an option for that? is that not necessary here?

Reply:
These docs suggest that this config option is on by default: https://prettier.io/docs/en/cli.html#--no-editorconfig

Reply:
ah yeah I tried to google it earlier but it wasn't clear at all 😓:+1: